inspectit-oce-core/src/test/java/rocks/inspectit/oce/core/config/FallbackConfigTest.java

```diff
@@ -0,0 +1,23 @@
+package rocks.inspectit.oce.core.config;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.TestPropertySource;
+import rocks.inspectit.oce.core.SpringTestBase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@TestPropertySource(properties = {
+        "inspectit.thread-pool-size=iAmNotANumber"
+})
+public class FallbackConfigTest extends SpringTestBase {
+
+    @Autowired
+    InspectitEnvironment env;
+
+    @Test
+    public void testFallbackConfigurationActive() {
```

Comment:
can you add a test that makes sure that when an invalid update occurs we remain with the previous config and not the fallback?